device/accton/x86_64-accton_as7712_32x-r0/preemphasis_init.soc

```diff
@@ -0,0 +1,802 @@
+echo "loading preemphasis..."
```

Comment:
@lguohan, unfortunately, config.bcm DOES NOT support the setting of TXFIR_POST2 / TXFIR_POST3.  That's why we put it within SOC.